### DISCUSSION TITLE
Github sponsor

### DISCUSSION THREAD

**Discussion by ghost** [community/community#63290] [Other Feature Feedback, Questions, & Ideas] [answered]
it's been 6 days but my Github sponsor application is still pending

**Comment by anjuan** [comment, marked as answer]
Hi, I'm the Engineering Manager for GitHub Sponsors. I'm trying it's taking us a while to review your application. Thank you for your patience. I hope we can get a response to you soon.

**Reply by MainaGitHUB** [reply]
Please review my GitHub sponsors payment. Was scheduled for Dec 22nd. Thank you.

**Comment by MahmoudAbdelsalamMahrous** [comment]
hello it has been a long time and my application is still pending 
I have been  waiting for about 6 months now

**Comment by Naveen-Pal** [comment]
@anjuan Same here, My application is still pending over few months.
Please look into it.
